api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -574,6 +574,7 @@ public Workspace setPublished(
   }
 
   @Override
+  @Transactional
```

Comment:
Try temporarily reverting this and adding `@Transactional(propagation = Propagation.NOT_SUPPORTED)` to WorkspaceServiceTest, see if that reproduces the issue.

Reply:
It reproduces the issue even if I keep this annotation.

Reply:
Unfortunate.. maybe, have you verified via manual testing whether this change fixes the issue?

Reply:
Still waiting for a server to spin up. This way it runs all the tests on someone else's computer